test/prompt/test_prompt_node.py

```diff
@@ -240,11 +241,15 @@ def test_simple_pipeline(prompt_model):
     Tests that a pipeline with a prompt node and prompt template has the right output structure
     """
     output_variable_name = "out"
-    node = PromptNode(prompt_model, default_prompt_template="sentiment-analysis", output_variable=output_variable_name)
+    node = PromptNode(
+        prompt_model, default_prompt_template="question-answering-per-document", output_variable=output_variable_name
```

Comment:
We can probably test using the same template,
but removing the `query` parameter from `pipe.run` call.
Am I missing something?